README.md

```diff
@@ -690,7 +690,9 @@ NOTE: To upload to the root of a drive, specify its root folder ID as the parent
 ardrive drive-info -d "c7f87712-b54e-4491-bc96-1c5fa7b1da50" | jq -r '.rootFolderId'
 ```
 
-By default, the single `upload-file` command will skip name conflicts found. To override this behavior and make a new revision of a file, use the `--replace` option:
+By default, the single `upload-file` command will `--upsert` on name conflicts found. This means that when it finds a file in the destination folder with the same name, it will compare the last modified dates of that file and the file to upload. If they are matching, the upload will be skipped, otherwise the upload will be added as a new revision.
```

Comment:
Don't forget to mention the "insert" part of "upsert"

Reply:
I've reworked this section, let me know what you think!